### PR TITLE
Draft: Hotkey list

### PR DESCRIPTION
**Pull request by serhiipylypchuk1991**

### CODE REVIEW

**Review by Sorokin-Oleg — APPROVED** [review]

File: docs/guides/diagram_editor/hot_keys.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+|---|---|
+|**Alt+1**|Allows you to hide/show Shapebar (*default mode only*)|
+|**Alt+2**|Allows you to hide/show Editbar|
+|**Alt+2**|Allows you to hide/show Grid Area|
+|**Ctrl+Z (CMD+Z)**|Allows you to revert the latest action|
+|**Ctrl+Shift+Z (CMD+Shift+Z)**|Allows you to return to the canceled action|
+|**Ctrl+D (CMD+D)**|Allows you to duplicate a selected element|
+|**Ctrl+C (CMD+C)**|Allows you to copy a selected element (*default mode only*)|
+|**Ctrl+V (CMD+V)**|Allows you to paste a selected element (*default mode only*)|
+|**Ctrl+Alt+C (CMD+Alt+C)**|Allows you to copy a style of the selected item (applicable, for elements of one essence)|
```
Comment (mafanya23) [inline docs/guides/diagram_editor/hot_keys.md:21]: 1) to copy the style; 
2) Нужна ли запятая после applicable? Возможно, essence => entity?

File: docs/guides/diagram_editor/hot_keys.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+|**Ctrl+Alt+V (CMD+Alt+V)**|Allows you to apply a copied style to the selected item (applicable, for elements of one essence)|
```
Comment (mafanya23) [inline docs/guides/diagram_editor/hot_keys.md:22]: Нужна ли запятая после applicable? Возможно, essence => entity?

File: docs/guides/diagram_editor/hot_keys.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+|**Ctrl+A (CMD+A)**|Allows you to select all items|
+|**Ctrl+Shift+A (CMD+Shift+A)**|Allows you to unselect all selected items|
+|**Ctrl+Mousewheel (CMD+Mousewheel)**|Allows you to increase/decrease a scale value|
```
Comment (mafanya23) [inline docs/guides/diagram_editor/hot_keys.md:25]: the scale value

File: docs/guides/diagram_editor/hot_keys.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+|**Shift+Left Click**|Allows you to add an item to the list of selected items|
+|**Alt+Left Click**|Allows you to unselect the selected item|
+|**Delete (Del)**|Delete an item(s)|
```
Comment (mafanya23) [inline docs/guides/diagram_editor/hot_keys.md:28]: Deletes?

**Review by mafanya23 — CHANGES_REQUESTED** [review]

**Review by mafanya23 — APPROVED** [review]

**Pull request merged**